src/index.js

```diff
@@ -20,6 +20,7 @@ Swagger.clearCache = clearCache
 Swagger.makeApisTagOperation = makeApisTagOperation
 Swagger.buildRequest = buildRequest
 Swagger.helpers = {opId}
+Swagger.baseUrl = baseUrl
```

Comment:
I'd like to rename the public-facing name for this function to `getBaseUrl`, for consistency with the existing public methods
```suggestion
Swagger.getBaseUrl = baseUrl
```